### ISSUE TITLE
Is this project still maintained?

### ISSUE THREAD

**Issue by Nstd** [wkhtmltopdf/wkhtmltopdf#5160]

**Comment by ashkulz** [comment]
Not really, see the [project status](https://wkhtmltopdf.org/status.html).

**Comment by Nstd** [comment]
> Not really, see the [project status](https://wkhtmltopdf.org/status.html).

Thanks for your efforts. ❤️

**Comment by promilo** [comment]
can anyone share alternatives to wkhtmltopdf?

**Comment by ashkulz** [comment]
@promilo there are some mentioned in the above link.

**Comment by Sicos1977** [comment]
https://github.com/Sicos1977/ChromeHtmlToPdf
